### PR TITLE
CI: Add ppc64le build to Travis

### PR DESCRIPTION
**Pull request by rbuch**

### CODE REVIEW

**Comment by evan-charmworks** [comment]
`/home/travis/build/UIUC-PPL/charm/pami-linux-ppc64le/bin/charmc: line 221: xlC_r: command not found`

**Comment by rbuch** [comment]
> `/home/travis/build/UIUC-PPL/charm/pami-linux-ppc64le/bin/charmc: line 221: xlC_r: command not found`

Yeah, I can't find a good source of information regarding what software they have available, so I'm just going to figure it out by looking at error messages. I suspect PAMI also won't be available, but I'm keeping it in the script just to find out for now.

**Review by matthiasdiener — APPROVED** [review]

**Review by evan-charmworks — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -24,3 +24,9 @@ jobs:
     script:
     - ./build all-test netlrts-linux-arm8 -j3 --with-production --enable-error-checking --enable-lbuserdata -g -Werror=vla || exit 1
     - make -C netlrts-linux-arm8/tmp test TESTOPTS="++local"
+  - os: linux
+    arch: ppc64le
+    name: "NetLRTS-Power"
+    script:
+    - ./build all-test netlrts-linux-ppc64le -j3 --with-production --enable-error-checking --enable-lbuserdata -g -Werror=vla || exit 1
+    - make -C netlrts-linux-ppc64le/tmp test TESTOPTS="++local"
```
Comment (matthiasdiener) [inline .travis.yml:32]:
```suggestion
    - ./buildold all-test pamilrts-linux-ppc64le gcc -j3 --with-production --enable-error-checking --enable-lbuserdata -g -Werror=vla || exit 1
    - make -C pamilrts-linux-ppc64le-gcc/tmp test TESTOPTS="++local"
```

Comment (matthiasdiener) [reply .travis.yml:32]: @rbuch Could we still test this with buidold? I think we currently have no pami* test at all.

Comment (evan-charmworks) [reply .travis.yml:32]: I think that configuration will still end up with xlC_r in CMK_NATIVE_CXX, which is not present on these machines.

Comment (rbuch) [reply .travis.yml:32]: In addition to not having the XL compilers installed, it's also unlikely that these machines have PAMI installed.

**Pull request merged**